### ISSUE TITLE
SA1513 incorrectly reported - brace followed by member access

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1173]
## Input

```csharp
bool contained =
    new[]
    {
        1,
        2,
        3
    }
    .Contains(3);
```

## Expected result

No report of SA1513.

## Actual result

SA1513 is reported.

**Comment by hvanbakel** [comment]
grabbed
